src/metrics.rs

```diff
@@ -45,4 +45,14 @@ impl Metrics {
         let hits = self.bloom_filter_hits.load(Relaxed) as f64;
         hits / queries
     }
+
+    /// Number of bloom filter queries performed.
+    pub fn bloom_filter_queries(&self) -> usize {
+        self.bloom_filter_queries.load(Relaxed)
+    }
+
+    /// Number of bloom filter hits (queries that avoided disk I/O).
+    pub fn bloom_filter_hits(&self) -> usize {
```

Comment:
`bloom_filter_negatives` or `io_skipped_by_filter` maybe; `hits` sounds like a `true positive`, but really the bloom filter is trying to find a `true negative`.

Reply:
updated the naming to be less confusing